src/EventStore.Core.Tests.XUnit/LogAbstraction/LogFormatAbstractorV3Tests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using EventStore.Core.LogAbstraction;
+using Xunit;
+
+namespace EventStore.Core.Tests.XUnit.LogAbstraction {
+	// check that lookups of the various combinations of virtual/normal/meta
+	// work in both directions and in the stream index.
+	// todo: ideally check that no unexpected (i.e. meta or virtual) streams have been
+	// added to the underlying index after each test.
```

Comment:
@timothycoleman can we start using xml comments to make this more ide friendly ?

Reply:
yeah fair enough for comments that describe an api. in this case though it's a test class so im not sure there is anywhere useful the xml comments would appear?

Reply:
I would normalize to one way of documenting classes and methods, will make things a bit easier.

Reply:
i could be persuaded on this, but perhaps generally the xmldoc is for a developer who's using the class's api (instantiating it, calling methods) and ordinary comments are for a developer who is looking at the code inside the class at implementation details, perhaps considering making a code change or hunting a bug